src/main/row.ts

```diff
@@ -210,45 +214,40 @@ function toHexString(byteArray: number[]): string {
  * objects are constructed for both the leads and the helpers
  * 
  * @param row Row to construct and retrieve BasecampTodoRequest objects for
- * @returns array of BasecampTodoRequest objects
+ * @returns a map associating role titles with BasecampTodoRequest objects
  */
-export function getBasecampTodoRequestsForRow(row: Row): BasecampTodoRequest[] {
-    let basecampTodoRequests: BasecampTodoRequest[] = [];
-    const leadsBasecampTodoRequest: BasecampTodoRequest | undefined = getBasecampTodoForLeads(row);
+export function getBasecampTodoRequestsForRow(row: Row): RoleRequestMap {
 
-    if(leadsBasecampTodoRequest !== undefined) {
-        basecampTodoRequests.push(leadsBasecampTodoRequest);
-    }
-
-    const helpersBasecampTodoRequest: BasecampTodoRequest[] = getBasecampTodosForHelpers(row);
+    const leadsRoleRequestMap: RoleRequestMap = getBasecampTodoForLeads(row);
+    const helperRoleRequestMap: RoleRequestMap = getBasecampTodosForHelpers(row);
 
-    if(helpersBasecampTodoRequest.length > 0) {
-        basecampTodoRequests = basecampTodoRequests.concat(helpersBasecampTodoRequest);
-    }
+    const allRoleRequestMap = {...leadsRoleRequestMap, ...helperRoleRequestMap};
```

Comment:
```suggestion
    const allRoleRequestMap: RoleRequestMap = {...leadsRoleRequestMap, ...helperRoleRequestMap};
```
Must have missed from before